examples/README.md

```diff
@@ -0,0 +1,38 @@
+# Examples
+## Modifying the default HiveMQ K8s Image Non-Root Image
```

Comment:
Would suggest: 
## Creating a custom HiveMQ k8s image with non-root user and group